pkg/filters/filter_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+package filters
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"storj.io/storj/pkg/storj"
+)
+
+var pieceIDs [][]byte
+var initDone bool
+var nbPiecesInFilter int
+var totalNbPieces int
+var falsePositiveProbability float64
+
+//  generates 1 million piece ids
+// adds 95% of them to the bloom filter,
+// and then checks all 1 million piece ids with the bloom filter
+// measure times, memory allocation, false positives
+
+func Init() {
+	if !initDone {
+		totalNbPieces = 1000000
+		nbPiecesInFilter = 950000
+		pieceIDs = GenerateIDs(totalNbPieces)
+		initDone = true
+		falsePositiveProbability = 0.1
+	}
+}
+
+func benchmarkAdd(filter Filter, pieceIDs [][]byte, b *testing.B) {
```

Comment:
I usually see that this kind of helper functions to get `b *testing.B` as a first parameter. 
Also add `b.Helper()` call.

These apply for the rest of the helper functions.